package.json

```diff
@@ -815,7 +815,7 @@
                 },
                 {
                     "command": "postgreSQL.connectDatabase",
-                    "when": "resourceLangId == sql"
+                    "when": "resourceLangId == psql"
```

Comment:
Any idea what the difference between resourceLangId and editorLangId is?

Reply:
I'm struggling to find docs on this. But as far as I can tell `resourceLangId` is used for context menus and `editorLangId` is used for command palette. I copied those properties from the existing mongo commands